crates/openapi/src/routes/payment_method.rs

```diff
@@ -192,3 +197,171 @@ pub async fn payment_method_delete_api() {}
     security(("ephemeral_key" = []))
 )]
 pub async fn default_payment_method_set_api() {}
+
+/// Payment Method - Create Intent
+///
+/// TODO: Description Here
```

Comment:
Please suggest the description here

Reply:
**Creates a payment method for customer with billing information and other metadata.**